upup/pkg/fi/cloudup/openstacktasks/instance.go

```diff
@@ -173,6 +173,7 @@ func (e *Instance) Find(c *fi.Context) (*Instance, error) {
 
 	ports, err := cloud.ListPorts(ports.ListOpts{
 		DeviceID: server.ID,
+		Tags:     fmt.Sprintf("%s=%s", openstack.TagClusterName, fi.StringValue(e.ServerGroup.ClusterName)),
```

Comment:
the problem that I see is that tags were added one year ago in kOps. If we have now clusters which are created before that, this is not going to work (because ports are missing tags).

Should we have way to rollback to old behaviour if we cannot find any port?

Reply:
> Should we have way to rollback to old behaviour if we cannot find any port?

Either that, or what about using the port name instead of the tag then?

Reply:
I thought about it and think it is a good idea to just fall back to the old behavior in case no port could be identified by tag.

Will code that later today.